python/py-slack-sdk/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        slackapi python-slack-sdk 3.1.1 v
+name                py-slack-sdk
+categories-append   irc
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     36 37 38 39
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         Python Slack SDK.
+long_description    The Slack platform offers several APIs to build apps. Each Slack API \
+                    delivers part of the capabilities from the platform, so that you can \
+                    pick just those that fit for your needs. This SDK offers a \
+                    corresponding package for each of Slack’s APIs. They are small and  \
+                    powerful when used independently, and work seamlessly when used \
+                    together, too. This package replaces py-slackclient.
+
+homepage            https://slack.dev/python-slackclient/
+
+checksums           rmd160  888f11df7d65660a5ce1cb3d7325b553cc273589 \
+                    sha256  858a8545c684ca5395a716dff7e9e379c242971242eb3e5161ebc154a300c37f \
+                    size    2996184
+
+if {${name} ne ${subport}} {
+    conflicts               py${python.version}-slackclient
```

Comment:
port lint picked up `Error: conflicts references unknown port: py39-slackclient`.

Maybe it's worth adding the py39 subport?